app/api/custom/events.py

```diff
@@ -56,6 +57,5 @@ def contact_organizer(event_id):
         html=attendee_mail.format(**context),
     )
     return jsonify(
-        status=True,
-        message="email is resend",
+        message="email sent to organizers",
```

Comment:
English sentences start with capital letters

How in JS am I going to know my request was successful?

Sachin, you created the notify endpoint, why don't you look there to see the expected response?

Reply:
sorry sir, i will always take care about this. sorry once again. i will fully check my commit before pushing.